docs/about.rst

```diff
@@ -88,3 +88,5 @@ Contributors
 * Tom Barclay
 * Rollin C. Thomas
 * Danny Goldstein
+
+ LocalWords:  sncosmo
```

Comment:
No, I didn't mean to add the LocalWords line.  [legacy of hitting 'A' once instead of just 'a' when running ispell]

Removed.

Added myself as contributor.  Pushed new commit to doc_typos to GitHub.  I believe this updates the Pull Request?

 - Michael

> On Jul 8, 2015, at 12:40 , Kyle Barbary <notifications@github.com> wrote:
> 
> In docs/about.rst:
> 
> > @@ -88,3 +88,5 @@ Contributors
> >  * Tom Barclay
> >  * Rollin C. Thomas
> >  * Danny Goldstein
> > +
> > + LocalWords:  sncosmo
> 
> Did you mean to add this line? Otherwise this PR looks good; thanks! (Oh, and feel free to add yourself to the contributors list while you're at it... every commit counts.)
> 
> —
> Reply to this email directly or view it on GitHub.
>